utils/husky-init.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const fs = require('fs');
+const shell = require('./shelljs');
+const packagePath = require('../utils/package-path')
+
+// This should be in each package's `prepare` script but we already use it to run `sf-install`.
+shell.exec('yarn husky install')
```

Comment:
Install `husky` after running `yarn install` in a package that uses `dev-scripts`.
Ideally this should be in each package's `prepare` script but since dev-scripts already uses it we run it here.

Bear in mind that while `.husky` folder is commited into a project, this commands generates the `husky.sh` file (which is always git-ignored) and move it into that folder. If the folder doesn't exist yet husky will create it.